test/src/search/strategies/reg_exp_test.dart

```diff
@@ -49,6 +49,27 @@ void main() {
       expect(result, expectedResult);
     });
 
+    test('Line start and end expressions work ', () {
+      const text = 'AaAa';
```

Comment:
Should this mean `^` and `$`?

Reply:
Yeah, I got distracted and forgot to finish it